AK/Statistics.h

```diff
@@ -53,8 +69,10 @@ class Statistics {
 
     T const median()
     {
-        quick_sort(m_values);
```

Comment:
Ah, here you delete this line. It's good enough for me, but if you want some practice with `git commit --fixup` and `git rebase -i`, this is your opportunity to learn.

Reply:
I'm deleting it in the commit where I actually replaced the median implementation.

Since that's essentially seperate from adding the algorithm in itself (since that can be used for other things than computing medians) I put them into seperate commits.